src/reservoir_lake.jl

```diff
@@ -1,5 +1,5 @@
-@get_units @with_kw struct SimpleReservoir{T}
-    Δt::T | "s"                                         # Model time step [s]
+@get_units @exchange @grid_type @grid_location @with_kw struct SimpleReservoir{T}
+    Δt::T | "s" | 0                                     # Model time step [s]
```

Comment:
Here you use `| 0` and in other places ` | 0 | "none" | "none"`. Is there a reason it isn't consistent? Either way is fine by me. If I understand correctly if exchange is 0, the grid type and var location should never be read anyway.